examples/event-source-mapping/main.tf

```diff
@@ -24,8 +24,9 @@ module "lambda_function" {
 
   event_source_mapping = {
     sqs = {
-      event_source_arn        = aws_sqs_queue.this.arn
-      function_response_types = ["ReportBatchItemFailures"]
+      event_source_arn                = aws_sqs_queue.this.arn
+      function_response_types         = ["ReportBatchItemFailures"]
+      maximum_concurrency_sqs_scaling = "2"
```

Comment:
this would become

```hcl
      scaling_config = {
        maximum_concurrency = 2
      }
```

Reply:
resolved in https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/402/commits/748c6c2d8a1152e020fd84ff3ed61d41d117314a